lib/multi.c

```diff
@@ -3043,3 +3046,8 @@ void Curl_multi_dump(struct Curl_multi *multi)
   }
 }
 #endif
+
+uint32_t Curl_multi_max_http2_concurrent_streams(struct Curl_multi *multi)
```

Comment:
inconveniently enough, we don't use `uint32_t` in our code (and this causes compiler error on some places). Stick to good old `unsigned int` and you'll get an unsigned 32 bit variable.

Reply:
Sure thanks, I had moved it to long, following other examples. Shall check on this.